modules/reference/pages/cluster-properties.adoc

```diff
@@ -489,6 +489,28 @@ When the disk usage of a node exceeds this threshold, it triggers Redpanda to mo
 
 ---
 
+=== write_caching
+
+The write caching mode to apply to a cluster. This caches batches of writes in memory until the segment appender chunk is full, instead of flushing for every `acks=all` write. This is the global default for all topics. It can be overridden with the topic property xref:topic-properties.adoc#writecaching[`write.caching`].
```

Comment:
lets remove all references to "segment appender chunk" thats a redpanda internal detail users don't understand.